libs/adler32cs.js

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2012 chick307 <chick307@gmail.com>
+ *
+ * Licensed under the MIT License.
+ * http://opensource.org/licenses/mit-license
+ */
+
+(function(jsPDF, callback) {
+  jsPDFAPI.adler32cs = callback();
```

Comment:
seems to be a mistake. 
Shouldnt it be 
`jsPDF.API.adler32cs`

?

Reply:
i could change it here and inside the jspdf call.

Reply:
I tested yesterday your adler32cs implementation and changed it like mentioned above ;)

Reply:
perfect